Create hr.svg

onyxfin opened the pull request:
# Description

Added Croatian language property file, Croatian flag .svg file.

Closes #(issue_number)

## Checklist:

- [ ] I have read the [Contribution Guidelines](https://github.com/Stirling-Tools/Stirling-PDF/blob/main/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings

## Contributor License Agreement

By submitting this pull request, I acknowledge and agree that my contributions will be included in Stirling-PDF and that they can be relicensed in the future under the MPL 2.0 (Mozilla Public License Version 2.0) license.

(This does not change the general open-source nature of Stirling-PDF, simply moving from one license to another license)

Frooodle commented on src/main/resources/static/images/flags/hr.svg:
```diff
@@ -0,0 +1 @@
+
```
This is not a valid svg

Frooodle commented:
PLease raise the changes as one single PR

onyxfin commented:
I have raised all under one PR #1345:

 

Added reference to the language in the navbar @ src/main/resources/templates/fragments/languages.html

 

Added croatian language property file (messages_hr_HR.properties) @ src/main/resources 

 

Added Croatian flag (hr.svg) @ src/main/resources/static/images/flags 

(downloaded from https://flagicons.lipis.dev/ )

 

From: Anthony Stirling ***@***.***> 
Sent: Thursday, May 30, 2024 10:53 PM
To: Stirling-Tools/Stirling-PDF ***@***.***>
Cc: onyxfin ***@***.***>; Author ***@***.***>
Subject: Re: [Stirling-Tools/Stirling-PDF] Create hr.svg (PR #1338)

 

PLease raise the changes as one single PR

—
Reply to this email directly,  <https://github.com/Stirling-Tools/Stirling-PDF/pull/1338#issuecomment-2140847313> view it on GitHub, or  <https://github.com/notifications/unsubscribe-auth/BIYH4UJSHTSH7OJTWBBVEKDZE6GUJAVCNFSM6AAAAABIRSTJKWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCNBQHA2DOMZRGM> unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>
